### PR TITLE
batch implementation for campaign/line item/targeting criteria

### PR DESCRIPTION
**Pull request by hwz**
implements batch support, as described in #82

### CODE REVIEW

File: twitter_ads/resource.py [hunk 1]
```diff
@@ -133,6 +134,50 @@ def _load_resource(self, klass, id, **kwargs):
...
+
+        resource = klass.BATCH_RESOURCE_COLLECTION.format(account_id=account.id)
+
+        json_body = []
+
+        for obj in objs:
+            entity_type = klass._ENTITY_MAP[klass.__name__].lower()
+            obj_json = {'params': obj.to_params()}
+
+            if obj.id is None:
```
Comment (jbabichjapan) [inline twitter_ads/resource.py]: Based upon how Python is so adept at handling dictionaries/json handling was trying to think of a way we could do this without the if/else blocks or logic here (risk for bugs)

Comment (tushdante) [inline twitter_ads/resource.py]: hey @jbabichjapan  how about replacing `if obj.id is None:` with `obj.get(id)` which will return `None` if `id` is not a key in `obj`

Comment (frewsxcv) [inline twitter_ads/resource.py]: Looking at the existing code, it looks like `obj.id` already has the possibility to return `None`?

Comment (frewsxcv) [inline twitter_ads/resource.py]: Nevermind. I'm now noticing there is a distinctino here between `id` existing with value `None` (not sure when this scenario happens) and an `id` not existing on the object.

**Pull request closed without merging**